meson.build

```diff
@@ -33,6 +33,7 @@ dir_libexec = get_option('prefix')/get_option('libexecdir')/'xkbcommon'
 cflags = [
     '-fno-strict-aliasing',
     '-Wno-unused-parameter',
+    '/wd4100', # MSVC C4100: unreferenced formal parameter
```

Comment:
can we move this up or down? It looks a bit jarring to me, moving it a line up would separate the warnings from the strict aliasing flag, or moving it to the bottom would group the `-W` and the "not `-W`" better.

Reply:
My intention was to keep it next to its gcc companion. But I can move it down.